contrib/valkey-go/option.go

```diff
@@ -27,7 +27,7 @@ func defaultConfig() *config {
 }
 
 // WithRawCommand can be used to set a tag `valkey.raw_command` in the created spans (disabled by default).
-// Warning: please note the datadog-agent currently does not support obfuscation for this tag, so use this at your own risk.
+// Warning: please note the datadog-agent v7.63.0 or below does not support obfuscation for this tag, so use this at your own risk.
```

Comment:
Quick nit; is the agent PR going out with v7.64.0 or will it go out in a patch release? If it is going out in v7.64.0, maybe we can change the version to say `v7.64.x` for more clarity.

Reply:
Updated as your suggestion!